tests/testthat/test_extract_date.R

```diff
@@ -0,0 +1,11 @@
+
+Context("Extract dates")
```

Comment:
```suggestion
context("Extract dates")
```

This was causing errors in the CI